docs/deploy.rst

```diff
@@ -0,0 +1,217 @@
+   | cluster  | cratedb.az1       | m      | A CrateDB cluster with standard storage                     | 0 - 2 (3 - 7 nodes)       |
+   | cluster  | cratedb.az1       | l      | A CrateDB cluster with standard storage                     | 0 - 3 (3 - 9 nodes)       |
+   | consumer | eventhub-consumer | xs     | Crate.io's Offering to connect Azure's EventHub to CrateDB. | 1 - 1 (1 - 1 instances)   |
+   | consumer | eventhub-consumer | s      | Crate.io's Offering to connect Azure's EventHub to CrateDB. | 1 - 3 (1 - 3 instances)   |
+   | consumer | eventhub-consumer | m      | Crate.io's Offering to connect Azure's EventHub to CrateDB. | 1 - 6 (1 - 6 instances)   |
+   | consumer | eventhub-consumer | l      | Crate.io's Offering to connect Azure's EventHub to CrateDB. | 1 - 18 (1 - 18 instances) |
+   +------------------------------------------------------------------------+-------------------+---------------------------+--------+
+
+This list shows your region's available CrateDB products. There are five tiers
+available for the deployment of CrateDB products: XS, S, M, L, XL.
+
+For this test deployment, pick tier ``xs`` which is the smallest available tier.
+According to the list above, this is the CrateDB cluster ``cratedb.az-gp1``.
+
+
+.. _launch-cluster:
+
+Launch the cluster
+==================
+
+Now that you've set up an organization and project, you can deploy your first
+cluster. Give this command:
+
+.. code-block:: console
+
+   sh$ croud clusters deploy \
+       --product-name cratedb.az-gp1 \
+       --tier xs \
+       --cluster-name my-first-crate-cluster \
+       --project-id 59c7961b-33a2-4478-9e98-2a6b3f54f60e \
+       --version 3.3.0 \
+       --username yourusername \
+       --password "yourpassword" \
+       --region bregenz.a1
+   +--------------------------------------+------------------------+----------+-------------+--------------------------------------+----------+------------------------------------------------------+
+   | id                                   | name                   | numNodes | crateVersion| projectID                            | username | fqdn                                                 |
+   +--------------------------------------+------------------------+----------+-------------+--------------------------------------+----------+------------------------------------------------------+
+   | 8d6a7c3c-61d5-11e9-a639-34e12d2331a1 | my-first-crate-cluster |        1 | 3.3.0       | 59c7961b-33a2-4478-9e98-2a6b3f54f60e | default  | my-first-crate-cluster.bregenz.a1.cratedb.net. |
+   +--------------------------------------+------------------------+----------+-------------+--------------------------------------+----------+------------------------------------------------------+
+  ==> Success: Cluster deployed. It may take a few minutes to complete the
+  changes.
+
```

Comment:
add a new para saying "Let's break that down:"

then add a list that goes over what was done step by step with that command